importAllData

```diff
@@ -4,4 +4,7 @@ set -u
 # Refresh source data, produce a data report and load the import mapping
 ./museum/importObjects -rpm
 # Run the import
-./museum/importObjects -i
+./museum/importObjects
+./museum/importSubjects -rm
```

Comment:
I've just stuck with individual comments that are actually relevant. I wasn't rebuilding the data report but it actually makes sense as there is a `SubjectsExport` view which should be included in the data report.